CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# v1.0.0
+
+## Features
+* feat: pass environment variables to cloud task via provider (Max Fechner)([29c3da9](https://github.com/lacework/terraform-azure-agentless-scanning/commit/29c3da96c5f2831a85b13dac6a2bee60c786b195))
+* feat: bump tf requirement to 1.5; add checks (#21) (Ao Zhang)([4c44fe8](https://github.com/lacework/terraform-azure-agentless-scanning/commit/4c44fe85e043c11f8d4fada33ce0f1bf0b0d75ee))
```

Comment:
This, to me, is both. A feature because we added checks (uhmm which for the user the feature is, extra validation, I would think), but it is a breaking change since we required Terraform >= 1.5

Can we call it out, similar to how we did it in this release (as an example) https://github.com/lacework/terraform-provider-lacework/releases/tag/v1.0.0